docs/integrations/destinations/mssql-v2.md

```diff
@@ -0,0 +1,12 @@
+# MSSQL (V2)
+
+## Changelog
```

Comment:
maybe easier to just point at the existing mssql docs? (plus I think this automatically publishes to the actual docs site, which we probably don't want)

Reply:
@edgao Fair enough.  I can delete this for now and just have it point at nothing.  I thought I remembered you having to create the changelog for iceberg to get things to work.

Reply:
iceberg-v2 is just using `/iceberg` for its doc link, so we're piggybacking on the existence of the old connector's changelog :P hacky, but it works (and it's what we did for s3v2)